payments-core/api/payments-core.api

```diff
@@ -3383,8 +3384,8 @@ public final class com/stripe/android/model/PaymentMethodOptionsParams$Card : co
 	public synthetic fun <init> (Ljava/lang/String;Ljava/lang/String;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
 	public final fun component1 ()Ljava/lang/String;
 	public final fun component2 ()Ljava/lang/String;
-	public final fun copy (Ljava/lang/String;Ljava/lang/String;)Lcom/stripe/android/model/PaymentMethodOptionsParams$Card;
-	public static synthetic fun copy$default (Lcom/stripe/android/model/PaymentMethodOptionsParams$Card;Ljava/lang/String;Ljava/lang/String;ILjava/lang/Object;)Lcom/stripe/android/model/PaymentMethodOptionsParams$Card;
+	public final fun copy (Ljava/lang/String;Ljava/lang/String;Ljava/lang/Boolean;)Lcom/stripe/android/model/PaymentMethodOptionsParams$Card;
+	public static synthetic fun copy$default (Lcom/stripe/android/model/PaymentMethodOptionsParams$Card;Ljava/lang/String;Ljava/lang/String;Ljava/lang/Boolean;ILjava/lang/Object;)Lcom/stripe/android/model/PaymentMethodOptionsParams$Card;
```

Comment:
Can we hide the moto parameter from the public api since it is only used from the createDashbaord function?

Reply:
oh, is this only happening because of that copy above?   Can PaymentMethodOPtionsParams.Card(moto = false) be called from outside the apk.  Seems like not?

Reply:
Yep, `moto` is only exposed via `copy()`.  You can test this in the `example` module code:

![Screen Shot 2021-12-17 at 10 13 27 AM](https://user-images.githubusercontent.com/91862945/146565841-a03144e4-5c7b-44c7-9020-61e7c93ec579.png)

Reply:
awesome, thanks for checking that out